tests/lib/mem_alloc/src/main.c

```diff
@@ -31,6 +31,12 @@
 #include <errno.h>
 #include <time.h>
 
+#ifdef __PTRDIFF_MAX__
+#define TOO_BIG __PTRDIFF_MAX__
```

Comment:
`__PTRDIFF_MAX__` is not standard and toolchain-specific.

`PTRDIFF_MAX`, which is part of the C standard from C99 and on, should be used.

Reply:
Thanks, I've updated the patch (without rebasing) to use PTRDIFF_MAX unconditionally after including stdint.h